examples/custom-fonts/README.md

```diff
@@ -10,7 +10,7 @@ Javascript example
 
 ## ♻️ BPMN Visualization Usage
 Override the BPMN element fonts using various ways. `mxGraph` knowledge is required to handle style changes.
-See https://process-analytics.github.io/bpmn-visualization-js/#bpmn-support-howto-elements-rendering for more information.
+See the [development documentation](https://github.com/process-analytics/bpmn-visualization-js/blob/master/docs/development/bpmn-support-how-to.md) for more details.
 
 :warning: In order to avoid to have to many content in the README, we simplify it. You can find all the content of the example in [custom-font.js](custom-font.js).
```

Comment:
```suggestion
:warning: In order to avoid having to many content in the README, we simplify it. You can find all the content of the example in [custom-font.js](custom-font.js).
```